frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/help-text.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { ExternalLink } from '@console/internal/components/utils';
+
+export const HelpTextCreator: React.FC<HelpTextCreatorProps> = (props) => {
+  const [show, setShow] = React.useState(true);
+  const { externalLink, title, Description } = props;
+  return (
+    show && (
+      <div className="co-noobaa-create-bc-info">
+        <div className="co-noobaa-create-bc-info-header">
+          <h2>{title}</h2>
+          <Button
+            className="co-noobaa-create-bc-info-header_close"
+            variant="plain"
+            aria-label="Action"
+            onClick={() => setShow(false)}
+          >
+            <TimesIcon />
+          </Button>
+        </div>
+        <div className="co-noobaa-create-bc-info-body">
+          <div className="co-noobaa-create-bc-info-body_description">
+            <Description />
+          </div>
+          {externalLink && <ExternalLink href={externalLink} text="Learn More" />}
+        </div>
+      </div>
+    )
+  );
+};
+
+export type HelpTextCreatorProps = {
+  title: string;
+  externalLink?: string;
+  Description: React.ComponentType;
```

Comment:
considering above change
```suggestion
  Description: React.ReactNode;
```